README.md

```diff
@@ -30,10 +30,15 @@ however the general workflow is the same.
 Clone this repo and navigate to the **voiceapp311/mycity/mycity/deploy_tools**
 directory and run the following command using Python 3:
 ```
+python deploy_tools.py -p -f <LAMBDA FUNCTION NAME HERE>
+```
+This will generate the lambda_function.zip archive and upload the code to your lambda function listed after the -f flag. Please replace everything after -f with your actual lambda function name (remove <> as well). Make sure you have aws command line installed prior to running the above command. If this command doesn't work then you can run the command
```

Comment:
Lets add a link to aws cli installation instructions: https://docs.aws.amazon.com/cli/latest/userguide/installing.html